test/services/optout/OptOutPropositionSpec.scala

```diff
@@ -99,7 +99,6 @@ class OptOutPropositionSpec extends UnitSpec {
 
     rowsWithoutHeaders.foreach(line => {
       val cells = line.split("\t").take(noOfColumns)
-//      println(cells.mkString("  --  "))
       println(s"(${
```

Comment:
looks like we still got **_println_** in our code base

Reply:
This is one of the weird files that isn't part of my code change. I think it needs looking at separately.